docs/modules/ROOT/pages/documentation/payara-server/server-configuration/certificate-management.adoc

```diff
@@ -106,6 +106,79 @@ asadmin> generate-self-signed-certificate --dn "CN=test.payara.fish,IP=192.168.1
 test_cert
 ----
 
+[[renew-self-signed-certificates]]
+== `renew-self-signed-certificates`
+
+Usage::
+`asadmin> renew-self-signed-certificates
+
+Aim::
+The renew-self-signed-certificates subcommand renews a self-certificate using the old distinguished name and alias.
+This certificate is then placed in the key and trust store of target instance or listener replacing the old one.
+
+[[command-options-self]]
+=== Command Options
+
+[cols="3,1,5,1,1",options="header"]
+|===
+|Option
+|Type
+|Description
+|Default
+|Mandatory
+
+|`--target`
+|String
+|The name of the instance to renew the certificate of.
```

Comment:
This is misleading. Instances have a keystore not specific certificates. Individual certificates are used for specific purposes (e.g. HTTP listener). Does this affect all certificates in the instance's keystore? Or just a specific certificate, e.g. admin HTTP listener certificate?